rellic/BC/Compat/IntrinsicInst.h

```diff
@@ -0,0 +1,17 @@
+#include "rellic/BC/Version.h"
+
```

Comment:
Include the LLVM `IntrinsicInst.h` header here, so that in place of including LLVM's intrinsic inst header, this header always gets included.